src/ui/locales/de.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Zuordnung umschalten",
+    "AttributionControl.MapFeedback": "Karten-Feedback",
+    "FullscreenControl.Enter": "Vollbildmodus aktivieren",
+    "FullscreenControl.Exit": "Vollbild beenden",
+    "GeolocateControl.FindMyLocation": "Finden Sie meinen Standort",
```

Comment:
```suggestion
    "GeolocateControl.FindMyLocation": "Finde meinen Standort",
```

I think this sentence is thought to be said by a human towards a machine? We don't say "Alexa, stellen Sie einen Timer", either.